.github/workflows/daphneci.yml

```diff
@@ -32,6 +32,8 @@ jobs:
         run: cargo fmt --all --check
       - name: Documentation
         run: RUSTDOCFLAGS='-D warnings' cargo doc --locked --no-deps --all-features --workspace
+      - name: Build
+        run: cargo build --all-targets --all-features
```

Comment:
why add this?

Reply:
CI was not building the benchmarks and did not detect a build error in daphne/benches/pine.rs.

Reply:
can you do only `--all-targets`? `--all-features` can cause weird issues where features that shouldn't be enabled together are enabled together

Reply:
Done.